src/main/java/supercoder79/ecotones/mixin/client/MixinBackgroundRenderer.java

```diff
@@ -27,5 +28,22 @@ private static void applyEcotonesFancyFog(float start) {
 //        } else {
 //            RenderSystem.setShaderFogStart(start);
 //        }
+//    }
+    
+    @ModifyArg(method = "applyFog", at = @At(value = "INVOKE", target = "Lcom/mojang/blaze3d/systems/RenderSystem;setShaderFogStart(F)V", ordinal = 1))
+    private static float applyEcotonesFancyFog(float start) {
+        if (ClientSidedServerData.isInEcotonesWorld) {
```

Comment:
@SuperCoder7979 This was commented out before with a TODO. Should I disable it for now?